bindata/network/ovn-kubernetes/managed/multi-zone-interconnect/ovnkube-control-plane.yaml

```diff
@@ -38,6 +38,7 @@ spec:
         type: infra
         openshift.io/component: network
         hypershift.openshift.io/control-plane-component: ovnkube-control-plane
+        hypershift.openshift.io/hosted-control-plane: {{.HostedClusterNamespace}}
```

Comment:
they should also have "hypershift.openshift.io/control-plane":true, both are used for our affinity opinions